lib/location/location.c

```diff
@@ -145,8 +145,10 @@ int location_request(const struct location_config *config)
 
 		if (config != NULL) {
 			/* Top level configs are given and must be taken from given config */
-			LOG_DBG("No method configuration given. "
-				"Using default method configuration.");
+
+			LOG_DBG("Storing received configuration as default.");
+			LOG_DBG("Interval = %d, Timeout = %d, Mode = %d",
+				config->interval, config->timeout, config->mode);
```

Comment:
All configuration parameters including these are logged in `location_core_config_log`:
```
mosh:~$ location get --interval 60 --timeout 30 --mode fallback
Started to get current location...
[00:21:15.747,467] <dbg> location: location_request: No method configuration given. Using default method configuration.
[00:21:15.747,528] <dbg> location: location_core_config_log: Location configuration:
[00:21:15.747,528] <dbg> location: location_core_config_log:   Methods count: 2
[00:21:15.747,558] <dbg> location: location_core_config_log:   Interval: 60
[00:21:15.747,558] <dbg> location: location_core_config_log:   Timeout: 30000ms
[00:21:15.747,589] <dbg> location: location_core_config_log:   Mode: 0
[00:21:15.747,589] <dbg> location: location_core_config_log:   List of methods:
[00:21:15.747,619] <dbg> location: location_core_config_log:     Method #0
[00:21:15.747,650] <dbg> location: location_core_config_log:       Method type: GNSS (2)
[00:21:15.747,680] <dbg> location: location_core_config_log:       Timeout: 120000ms
[00:21:15.747,741] <dbg> location: location_core_config_log:       Accuracy: normal (1)
[00:21:15.747,741] <dbg> location: location_core_config_log:     Method #1
[00:21:15.747,772] <dbg> location: location_core_config_log:       Method type: Cellular (1)
[00:21:15.747,802] <dbg> location: location_core_config_log:       Timeout: 30000ms
[00:21:15.747,833] <dbg> location: location_core_config_log:       Service: Any (0)
[00:21:15.747,863] <dbg> location: location_core_config_log:       Cell count: 4
[00:21:15.747,924] <dbg> location: location_core_location_get_pos: Requesting location with 'GNSS' method
```